doc/api/url.md

```diff
@@ -230,11 +230,28 @@ manner similar to that of a Web browser resolving an anchor tag HREF.
 For example:
 
 ```js
-url.resolve('/one/two/three', 'four')         // '/one/two/four'
-url.resolve('http://example.com/', '/one')    // 'http://example.com/one'
-url.resolve('http://example.com/one', '/two') // 'http://example.com/two'
+url.resolve('/one/two/three', 'four');         // '/one/two/four'
+url.resolve('http://example.com/', '/one');    // 'http://example.com/one'
+url.resolve('http://example.com/one', '/two'); // 'http://example.com/two'
 ```
 
+Some protocols (`http`, `https`, `ftp`, `gopher` and `file`) are treated as special cases.
```

Comment:
Can you add a line break at column <= 80? This applies for all lines not only this one.
Thanks!